### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by anandsr21harshit**
Navigation bar added

### CODE REVIEW

File: Components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.nav-bar{
+    background-color: tomato;
```
Comment (OmKakatkar) [inline Components/Navigation/navigation.css]: Try to use variables for colors, they help if you need to change the color later on

File: Components/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.menu{
+    list-style: none;
+    margin-right: 1rem;
+}
+
+.menu > a{
+    text-decoration: none;
+    color: white;
```
Comment (OmKakatkar) [inline Components/Navigation/navigation.css]: CSS variables for color

**Review by OmKakatkar — CHANGES_REQUESTED** [review]

File: Components/Navigation/navigation.css (hunk 1 above)
Comment (anandsr21harshit) [reply Components/Navigation/navigation.css]: Done

File: Components/Navigation/navigation.css (hunk 2 above)
Comment (anandsr21harshit) [reply Components/Navigation/navigation.css]: Done

**Review by OmKakatkar — APPROVED** [review]

**Review by romabulani — APPROVED** [review]
Good to merge

**Pull request merged**